recipes/einsums/meta.yaml

```diff
@@ -0,0 +1,106 @@
+{% set name = "Einsums" %}
+{% set version = "0.2" %}
+{% set sha256 = "6f0e209add31b38d58a557e14afce7b23d5dd684a70aaf15554469f19ce77d29" %}
+{% set build_number = 0 %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/{{ name }}/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: {{ build_number }}
+  string: {{ blas_impl }}_h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}
+  skip: true                                                               # [win]
+  binary_relocation: true
+  run_exports:
+    - {{ pin_subpackage('einsums', max_pin='x.x') }}
+  ignore_run_exports:
+    - elfutils                                                             # [linux]
+    - libzlib
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+    - ninja
+    - llvm-openmp
+  host:
+    - catch2
+    - elfutils                                                             # [linux]
+    - fmt 10.1
+    - hdf5
+    - range-v3 0.12
+    - zlib
+    - mkl-devel {{ mkl }}                                                  # [blas_impl == "mkl"]
+    - blas-devel * *_{{ blas_impl }}                                       # [blas_impl == "openblas"]
+    - fftw                                                                 # [blas_impl == "openblas"]
```

Comment:
Is this meant to be constrained by `openblas`?

Reply:
Are you asking about fftw in particular being constrained by openblas? In that case, yes. einsums uses the internal FFT when MKL is selected, and MKL & OpenBLAS are the only implementations allowed at present.

Thanks for the review! Sorry I wasn't monitoring this PR last week. I'll fix up the OpenMP items you raised.